### PR TITLE
Updates decisions to DfE-preferred nomenclature. Updates architecture …

### PR DESCRIPTION
**Pull request by michael-fielding**
…diagrams due to latest changes

### CODE REVIEW

File: documentation/architecture/achecitecture-decision-records/data-storage.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## Considered Options
+- Azure Cosmos DB, a fully-managed and distributed NoSQL database
+- Azure Managed SQL, a fully-managed and serverless distributed SQL database
+- Azure Blob Storage, an object storage solution
+- Azure Synapse, a Data Warehousing solution
+
+### Evaluation
+
+| Criteria | Comment | Cosmos DB | SQL | Blob Storage | Synapse |
```
Comment (Faizi52996) [inline documentation/architecture/achecitecture-decision-records/data-storage.md:20]: the evaluation table doesn't seems to align correctly on my end. attaching image for  reference. 
image

Comment (Faizi52996) [reply documentation/architecture/achecitecture-decision-records/data-storage.md:20]: There is space missing after after total column. Image for reference attached. 
image

Comment (michael-fielding) [reply documentation/architecture/achecitecture-decision-records/data-storage.md:20]: Not sure why you're seeing that, as I see this:
Screenshot 2024-01-12 095503

Comment (Faizi52996) [reply documentation/architecture/achecitecture-decision-records/data-storage.md:20]: thats strange. I could still see the old way and adding a space to it fixes it for me. if adding a space doesn't break it for you then would be good to add it.

Comment (michael-fielding) [reply documentation/architecture/achecitecture-decision-records/data-storage.md:20]: @faizi I have added the spaces

**Review by Faizi52996 — APPROVED** [review]

**Pull request merged**